### DISCUSSION TITLE
How do I get around docker images failing to download during installation?

### DISCUSSION THREAD

**Discussion by krejac** [telekom-security/tpotce#1271] [Q&A] [answered]
I have read (and understood) the  [Docker Images Fail to Download](https://github.com/telekom-security/tpotce#docker-images-fail-to-download)-part of the readme but I'm wondering how to apply this _during_ installation... Any advise here?

**Comment by t3chn0m4g3** [comment, marked as answer]
The only solution at the moment is to use the post install method which allows you to run `docker login` before the installation and then the quota from your docker account applies.

**Comment by krejac** [comment]
Thanks I'll test that out before deploying in production. 👍
